src/SmartComponents/RunTaskModal/SystemsSelect.js

```diff
@@ -32,6 +35,19 @@ const SystemsSelect = ({
     .replace(/-/g, '_')}_WARNING`;
 
   const [filterSortString, setFilterSortString] = useState('');
+  const [validated, setValidated] = useState('default');
+  const [helperText, setHelperText] = useState(null);
+
+  const handleSetTaskName = (_event, name) => {
+    setTaskName(name);
+  };
+
+  useState(() => {
+    if (Object.hasOwn(createTaskError, 'status')) {
+      setHelperText(createTaskError.statusText);
+      setValidated('error');
+    }
+  }, [createTaskError.statusText]);
```

Comment:
The dependency should just be `createTaskError`, since also other props of the object are used in here.